conda/recipes/libcudf/meta.yaml

```
@@ -37,20 +37,34 @@ requirements:
     - cmake {{ cmake_version }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
     - librmm ={{ minor_version }}
     - libkvikio ={{ minor_version }}
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    - libcufile {{ cuda11_libcufile_host_version }}  # [linux64]
+    - libcufile-dev {{ cuda11_libcufile_host_version }}  # [linux64]
+    - libcurand {{ cuda11_libcurand_host_version }}
+    - libcurand-dev {{ cuda11_libcurand_host_version }}
     - cuda-nvrtc ={{ cuda_version }}
```

Do we want CUPTI in the CUDA 11 case (like is the case in CUDA 12)?

```suggestion
    - cuda-cupti ={{ cuda_version }}
    - cuda-cupti-dev ={{ cuda_version }}
    - cuda-nvrtc ={{ cuda_version }}
```

---

No, I actually removed the cupti dependency from our builds in #13528. It was needed by NVBench but I disabled that option since it breaks minor version compatibility (e.g. if built with 11.8, the benchmarks couldn't be run on any other minor version). I deleted CUPTI from the CUDA 12 dependencies in 9590817.